src/js/pagination/paging-collection.js

```diff
@@ -48,6 +48,8 @@
                 pageSize: 'page_size',
                 sortKey: 'order_by',
                 order: 'sort_order',
+                totalPages: null,
```

Comment:
yeah exactly, we don't want this. But they are included in the parent's `queryParams`. So every thing that is truthy in query params gets included in the query params. By specifically marking them null ensures that they won't include in the final query params. 
`totalRecords` and `totalPages` are the `state` attributes and they the value of these depends on what we have in queryParams. So e.g. if we have the following queryParams the value of `totalRecords` and `totalPages` will depend on the response's `count` and  `total_pages `key respectovely.

```javascript
queryParams: {
    totalRecords: 'count',
    totalPages: 'total_pages'
}
```

PageableCollection assumes that the `totalRecords = response's 'total_entries' and totalPages = 'total_pages' `. And we can override this `queryParams` object.